Makefile

```diff
@@ -67,6 +67,11 @@ deploy: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+release-artifacts: manifests kustomize
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
```

Comment:
Should this step be factored out into a separate make target? Used above aswell.